layers/+lang/zig/README.org

```diff
@@ -6,14 +6,43 @@
 - [[#description][Description]]
   - [[#features][Features:]]
 - [[#install][Install]]
+  - [[#layer][Layer]]
+  - [[#lsp-zls][LSP (zls)]]
+- [[#key-bindings][Key bindings]]
+  - [[#lsp][LSP]]
 
 * Description
 This layer adds support for the zig programming language [[https://ziglang.org/]]
 
 ** Features:
 - Syntax Highlighting
+- LSP support via =zls= (see [[#lsp-zls][=zls=]])
```

Comment:
A local link in org will not work when HTML is generated from the org files, please link to the develop.spacemacs.org website instead.

Reply:
I took this from the [README of the Go Layer](https://github.com/syl20bnr/spacemacs/blob/develop/layers/%2Blang/go/README.org) which has a local link ("`see LSP backend`") and it [seems to work also in the website](https://develop.spacemacs.org/layers/+lang/go/README.html#lsp-gopls).

Reply:
Hmm good point, I have thought this would not be supported but it looks like I have been proven wrong, lets keep it then.